lib/ogsf/gvl_calc.c

```diff
@@ -721,6 +721,8 @@ int gvl_isosurf_calc(geovol *gvol)
     }
 
     /* TODO: G_free() dbuff and need_update ??? */
+    G_free(dbuff);
+    G_free(need_update);
```

Comment:
```suggestion
    G_free(dbuff);
    G_free(need_update);
```
The TODO can now be removed.

Reply:
Done